src/book/03-guides/03-hardware/03-batteries.mdx

```diff
@@ -0,0 +1,86 @@
+   adaptor lead, shown below.
+
+3. Once both cables on the battery are hooked up, the robot can safely be turned on. Be sure to carefully load the battery into the robot to avoid damage, this may
+   require some technique so don't be afraid to ask a team member for the best way to go about this.
+
+![Battery in robot with a battery monitor](./images/batteries/battery_in_robot.jpg)
+
+## Charging
+
+We have two types of chargers in the lab to charge LiPo batteries. Both achieve the same purpose and have a similar user interface. The Powertech Plus charger will be
+used for the purposes of the images below.
+
+### Charging a battery
+
+1. Plug the battery cable with the white plug into the corresponding socket on the side of the charger
+
+2. Plug the other battery cable into the main output cable of the charger, this may require an adaptor, as seen in the section on using in the robot
+
+3. Plug the Powertech Plus into a power socket, and use the arrow buttons to select the "Charging" setting for the type of battery you a charging. For the NUgus robot
+   battery this is a 14.8V 4 Cell, as seen in the image below. For a Darwin robot battery, this is a 11.1V 3 Cell.
+
+4. Hold down the start button until the device beeps, and observe the content on the screen. If the charging device indicates any errors, stop charging immediately,
+   unplug the battery, and return it to a fire retardant case. If there are no errors, press start to commence charging
+
+5. Close and lock the white BAT-SAFE box with the battery inside (Keep the charger outside if possible to hear when it is done)
+
+6. Stay in the lab with the battery and disconnect immediately when charge has completed. Never leave a battery charging when there is no one in the lab
+
+![Charging setting on Powertech Plus LiPo Charger](./images/batteries/charge.jpg)
+
+## Storing and Traveling with batteries
+
+When travelling, LiPo batteries should always be kept in fire retardant bags, with one battery stored in each fire retardant bag. If travelling on an aircraft,
+LiPo batteries should always go in carry-on luggage, never in checked luggage. It is important to prepare batteries for storage when travel in aircraft. Batteries
+should also be prepared for storage when being stored in the lab for an extended period of time.
+
+In both cases, preparing a battery for storage is done by following the same process outlined for charging, however by using the "Storage" setting, as seen in the
+image below.
+
+![Storage setting on Powertech Plus LiPo Charger](./images/batteries/storage.jpg)
+
+## Disposal of batteries
+
+If a battery has been physically damaged or is showing errors when attempting to charge, it should be disposed of. Batteries for disposal should be taken to the
+Electrical Engineering building on campus. Talk to the team leader to confirm the most up to date details of where to take them.
```

Comment:
Acquiring some electrical tape so you can "bandage" damaged cables (so they dont short out when in the fire retardant bag might be a good idea too)